libs/utils/android/system.py

```diff
@@ -23,15 +23,16 @@
 import pexpect as pe
 
 GET_FRAMESTATS_CMD = 'shell dumpsys gfxinfo {} > {}'
+SYSTRACE_EVENTS_DEFAULT = ['gfx', 'view', 'sched', 'freq', 'idle']
+SYSTRACE_BUFFSIZE_DEFAULT = 10240
 
 class System(object):
     """
     Collection of Android related services
     """
 
     @staticmethod
-    def systrace_start(env, trace_file, time=None,
-                       events=['gfx', 'view', 'sched', 'freq', 'idle']):
+    def systrace_start(env, trace_file, time=None):
```

Comment:
ATM `systrace_start()` is only used in workload.py by `tracingStart()`, and the call doesn't specify events - for now it seems all the tracing done with `collect='systrace'` would just use the default events. So unless I'm mistaken, it might be okay to remove `events`, which is redundant anyway with the configuration dict.